lib/matplotlib/dates.pyi

```diff
@@ -0,0 +1,200 @@
+import datetime
+import numpy as np
+from typing import Any, Callable, Dict, List, Sequence
+
+from .axis import Axis
+from .projections.polar import _AxisWrapper
+from .ticker import _DummyAxis, Formatter, Locator
+from .units import AxisInfo, ConversionInterface
+
+
+# --- Global Functions ---
+
+def _get_tzinfo(tz: str | datetime.tzinfo | None = ...) -> datetime.tzinfo: ...
+
+def _reset_epoch_test_example() -> None: ...
+
+def set_epoch(epoch: str) -> None: ...
+
+def get_epoch() -> str: ...
+
+def _dt64_to_ordinalf(d: np.datetime64 | np.ndarray) -> float | np.ndarray: ...
```

Comment:
There seems to be a bug here; it doesn't actually accept `np.datetime64` or return a `float`, but crashes. Maybe something to be fixed, or else the type hint shouldn't mention them.